client/css/style.css

```diff
@@ -2792,3 +2790,61 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 #chat table.channel-list .topic {
 	white-space: pre-wrap;
 }
+
+/* Jump to feature WIP styles */
```

Comment:
The whitespace hack was deliberately the last style in css.

We should use `@import` already :)

Reply:
Will move the styles to the component itself :+1: